applications/nrf_desktop/Kconfig.sysbuild

```diff
@@ -1,31 +0,0 @@
-#
```

Comment:
Should you appear under both `Co-authored-by` and `Signed-off-by` in the commit message?

Reply:
`Co-authored-by` is GitHub feature that shows that this commit has multiple authors and `Signed-off-by` is a requirement so the compliance passes.
Probably I could remove `Signed-off-by` and still satisfy compliance but as is also should work.
We will check it after I'll remove the draft status.

Reply:
Thanks for clarification